src/console/prompt.ts

```diff
@@ -70,14 +95,26 @@ export function innerPrompt(
 interface DrawState {
   title: string;
   inputText: string;
+  mask: InputMask | false;
   hasCompleted: boolean;
 }
 
 function render(state: DrawState): TextItem[] {
+  let { inputText } = state;
+  if (state.mask) {
+    const char = state.mask.char ?? defaultMask.char;
+    const lastVisible = state.mask.lastVisible ?? defaultMask.lastVisible;
```

Comment:
I just realized that this probably needs to check `state.hasCompleted` so that this clears the last character when doing:

```
import $ from "https://raw.githubusercontent.com/andrewbrey/dax/andrewbrey/feat/input-mask/mod.ts"

$.prompt("Testing", {
  mask: {
    lastVisible: true,
  },
  noClear: true,
});
```

Would you be able to update this and add a test? Thanks and sorry I didn't notice last time.